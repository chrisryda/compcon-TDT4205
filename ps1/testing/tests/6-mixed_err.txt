dx= // Comment
go go goooo
dy=11dx=11
-5: go
go / Comment
8: dy=42 dx=4.2
I beheld the wretchâ€”the miserable monster whom I had created
apple pie
