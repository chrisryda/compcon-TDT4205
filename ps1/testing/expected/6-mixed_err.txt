Table filled!
line    1: error: dx= // Comment
~~~~~~~~~~~~~~~~~~~~~^
line    2: error: go go goooo
~~~~~~~~~~~~~~~~~~~~~~~~~~^
line    3: error: dy=11dx=11
~~~~~~~~~~~~~~~~~~~~~~~^
line    4: error: -5: go
~~~~~~~~~~~~~~~~~~^
line    5: error: go / Comment
~~~~~~~~~~~~~~~~~~~~~~^
line    6: error: 8: dy=42 dx=4.2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
line    7: error: I beheld the wretchâ€”the miserable monster whom I had created
~~~~~~~~~~~~~~~~~~^
line    8: error: apple pie
~~~~~~~~~~~~~~~~~~^
